systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleInstallation.java

```diff
@@ -0,0 +1,404 @@
+        KubeResourceManager.get().createResourceWithWait(
+            RoleTemplates.roleFromFile(namespaceToWatch,
+                LeaseUtils.changeLeaseNameInResourceIfNeeded(roleFile.getAbsolutePath(), clusterOperatorConfiguration.getExtraEnvVars())
+            )
+        );
+
+        roleFile = new File(TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/023-ClusterRole-strimzi-cluster-operator-role.yaml");
+        roleFile = RbacUtils.switchClusterRolesToRolesIfNeeded(roleFile, clusterOperatorConfiguration.isNamespaceScopedInstallation());
+        KubeResourceManager.get().createResourceWithWait(
+            RoleTemplates.roleFromFile(namespaceToWatch, roleFile.getAbsolutePath())
+        );
+
+        roleFile = new File(TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/030-ClusterRole-strimzi-kafka-broker.yaml");
+        roleFile = RbacUtils.switchClusterRolesToRolesIfNeeded(roleFile, clusterOperatorConfiguration.isNamespaceScopedInstallation());
+        KubeResourceManager.get().createResourceWithWait(
+            RoleTemplates.roleFromFile(namespaceToWatch, roleFile.getAbsolutePath())
+        );
+
+        roleFile = new File(TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/031-ClusterRole-strimzi-entity-operator.yaml");
+        roleFile = RbacUtils.switchClusterRolesToRolesIfNeeded(roleFile, clusterOperatorConfiguration.isNamespaceScopedInstallation());
+        KubeResourceManager.get().createResourceWithWait(
+            RoleTemplates.roleFromFile(namespaceToWatch, roleFile.getAbsolutePath())
+        );
+
+        roleFile = new File(TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/033-ClusterRole-strimzi-kafka-client.yaml");
+        roleFile = RbacUtils.switchClusterRolesToRolesIfNeeded(roleFile, clusterOperatorConfiguration.isNamespaceScopedInstallation());
+        KubeResourceManager.get().createResourceWithWait(
+            RoleTemplates.roleFromFile(namespaceToWatch, roleFile.getAbsolutePath())
+        );
+    }
+
+    private void applyClusterRoleBindings() {
+        // 021-ClusterRoleBinding
+        ClusterRoleBindingResource.clusterRoleBinding(clusterOperatorConfiguration.getNamespaceName(),
+            TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/021-ClusterRoleBinding-strimzi-cluster-operator.yaml");
+        // 030-ClusterRoleBinding
+        ClusterRoleBindingResource.clusterRoleBinding(clusterOperatorConfiguration.getNamespaceName(),
+            TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/030-ClusterRoleBinding-strimzi-cluster-operator-kafka-broker-delegation.yaml");
+        // 033-ClusterRoleBinding
+        ClusterRoleBindingResource.clusterRoleBinding(clusterOperatorConfiguration.getNamespaceName(),
+            TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator/033-ClusterRoleBinding-strimzi-cluster-operator-kafka-client-delegation.yaml");
+    }
```

Comment:
```suggestion
private void applyClusterRoleBindings() {
    String basePath = TestConstants.PATH_TO_PACKAGING_INSTALL_FILES + "/cluster-operator";
    List<String> bindings = List.of(
        "021-ClusterRoleBinding-strimzi-cluster-operator.yaml",
        "030-ClusterRoleBinding-strimzi-cluster-operator-kafka-broker-delegation.yaml",
        "033-ClusterRoleBinding-strimzi-cluster-operator-kafka-client-delegation.yaml"
    );

    for (String fileName : bindings) {
        ClusterRoleBindingResource.clusterRoleBinding(
            clusterOperatorConfiguration.getNamespaceName(),
            new File(basePath, fileName).getAbsolutePath()
        );
    }
}
```

Reply:
Same as above, but you are right that I shouldn't use the `ClusterRoleBindingResource` class :) good catch.